QUICKSTART.md

```diff
@@ -1,24 +1,6 @@
 # Quick Start Guide
 
-This document provides the steps to create a quick setup on a local Mac OS or Linux host using a cluster of Virtual Box VMs setup using vagrant.
-
-## Pre-requisites
-
-* [Install Virtual Box 5.1.14 or later]( https://www.virtualbox.org/wiki/Downloads )
-* [Install Vagrant 1.9.1 or later]( https://www.vagrantup.com/downloads.html )
-* [Install Docker 1.12 or later]( https://docs.docker.com/engine/installation/ )
-* Clone the Contiv install repository <br>
-`git clone http://github.com/contiv/install'
-
-## Setup the cluster with Contiv for Kubernetes
-`make demo-k8s`
-
-## Setup the cluster with Contiv for Docker with Swarm
-`make demo-swarm`
-
-## Customizing the setup
-
-* The default configuration creates a 2 node cluster. To increase the number of nodes set the environment variable `CONTIV_NODES=<n>`
+Please follow the tutorials [here](http://contiv.github.io/documents/tutorials/container-101.html).
```

Comment:
Wouldnt this be better link to point : http://contiv.github.io/documents/tutorials/ ?
So that use can choose whatever they want to learn ?